packages/material-icons/.yo-rc.json

```diff
@@ -1,6 +1,6 @@
 {
   "generator-react-native-vector-icons": {
-    "fontName": "MaterialIcons-Regular",
+    "fontName": "Material Icons",
```

Comment:
@vonovak Is this right? if I run fc-query on the font I get `MaterialIcons-Regular`
I think I might also modify the generator to call this postscriptName of split it if it's being used in multiple places